Code/GraphMol/ChemReactions/ReactionParser.h

```diff
@@ -148,6 +148,26 @@ inline std::string ChemicalReactionToRxnSmiles(const ChemicalReaction &rxn,
   params.canonical = canonical;
   return ChemicalReactionToRxnSmiles(rxn, params);
 }
+
+//! returns the reaction SMARTS for a reaction with CX extension
+RDKIT_CHEMREACTIONS_EXPORT std::string ChemicalReactionToRxnCXSmarts(const ChemicalReaction &rxn,
```

Comment:
It seems to me like this might be better as `ChemicalReactionToCXRxnSmarts()` (and then `ChemicalReactionToCXRxnSmiles()` below).

My logic: if I want to add CX extensions to SMARTS I call `MolToCXSmarts()`, so if I want to add CX extensions to Reaction SMARTS I would call `ChemicalReactionToCXRxnSmarts()`.

I can see arguments both ways, so if you really prefer it this way we can keep it.

Reply:
Sounds good!